app/models/protocol.rb

```diff
@@ -260,6 +266,10 @@ def active?
     study_type_question_group.nil? ? false : study_type_question_group.active
   end
 
+  def has_human_subject_info?
```

Comment:
ActiveRecord provides `#human_subject_info?`, so I think this method is a little redundant.